server/database/migrations/20190528142514_roles.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('roles', (table) => {
+    table.increments();
+    table
+      .string('roleName', 30)
+      .notNull()
+      .unique();
+    table.string('canBuy', 100).notNull();
+    table.string('canSell', 100).notNull();
```

Comment:
Same as above... I think the link I posted doesn't work, but something like

table.boolean('canBuy').notNull();